mix.exs

```diff
@@ -43,7 +43,7 @@ defmodule Stripe.Mixfile do
   defp env do
     [
       api_base_url: "https://api.stripe.com",
-      api_upload_url: "https://files.stripe.com/v1/",
+      api_upload_url: "https://files.stripe.com",
```

Comment:
confirmed, we https://github.com/beam-community/stripity-stripe/blob/73faf6d3e4b021471db7f2768cf2fbdd69055557/lib/generated/file.ex#L142